### PR TITLE
ponderm created their first post

### PR DESCRIPTION
**Pull request by ponderm**
This is a post for ponderm.

### CODE REVIEW

File: _posts/ponderm/2018-05-17-ponderm-first-post.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+author: ponderm
+title: "ponderm's first post!"
+---
+Hi, my name is Maddy. I am a Professional Science Master's student in the Biomedical & Health Informatics program.
```
Comment (eah13) [inline _posts/ponderm/2018-05-17-ponderm-first-post.md]: Hello and welcome!  nice first post.

**Comment by ponderm** [comment]
Sweet 🍬

File: _posts/ponderm/2018-05-17-ponderm-first-post.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+author: ponderm
+title: "ponderm's first post!"
```
Comment (ponderm) [inline _posts/ponderm/2018-05-17-ponderm-first-post.md]: Congrats! 🎊

**Comment by ponderm** [comment]
If you would like to see a vampire in code, go here [https://trinket.io/python/0a12f6dde1](url).

**Pull request merged**